src/inspect_ai/_cli/eval.py

```diff
@@ -399,6 +399,12 @@ def eval_command(
     help="Do not cleanup failed log files after retries",
     envvar="INSPECT_EVAL_NO_RETRY_CLEANUP",
 )
+@click.option(
+    "--view-bundle-dir",
```

Comment:
I think this might need to be `--bundle-dir` to match the arg?

Reply:
whoops thank you